nui-input/src/main/java/org/terasology/input/device/MouseDevice.java

```diff
@@ -34,19 +35,21 @@
     /**
      * @return The current position of the mouse in screen space
      */
-    Vector2i getMousePosition();
+    Vector2i getPosition();
 
     /**
      * @return The change in mouse position over the last update
      */
-    Vector2i getDelta();
+    Vector2d getDelta();
```

Comment:
(probably unrelated to this change) Should these getters return the modifiable vectors or the constant interfaces instead?